pom.xml

```diff
@@ -265,6 +269,12 @@
             <version>1.3.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.assertj</groupId>
+            <artifactId>assertj-core</artifactId>
+            <version>3.13.2</version>
```

Comment:
can you extract artifact version as property? 
Do we really need this library? I do not see that we are using it other projects.

Reply:
done

Reply:
this lib is already part of spring-boot-starter-test. I want to use newer version. But lib itself is very cool.